spec/file_database_spec.rb

```diff
@@ -65,6 +65,36 @@
     end
   end
 
+  describe '#contact_at' do
+    it 'takes an index and returns the contact in that index' do
+      database.create(test_details)
+      database.create(second_contact)
+
+      expect(database.contact_at(0)).to eq(test_details)
+    end
```

Comment:
Is a good practice (when possible) to not use other methods of the class you are testing in the test of a method, so that the tests are truly independent from each other.

For example, if something changed in the implementation of `create`, you don't want the test for `contact_at` to break, as it is unrelated.

Instead, use your method `file.write(create_json_contact)`